addons/editor/stringtable.xml

```diff
@@ -73,15 +73,19 @@
         </Key>
         <Key ID="STR_ZEN_Editor_EjectPassengers">
             <English>Eject Passengers</English>
+            <German>Passagiere rauswerfen</German>
         </Key>
         <Key ID="STR_ZEN_Editor_EjectPassengers_Description">
             <English>Ejects the passengers of all selected vehicles. If the vehicle is currently flying, the units will be given parachutes.\nUnlike vanilla eject, units will disembark one after another (not immediately, with a short delay in-between).</English>
+            <German>Wirft die Fahrgäste aller ausgewählten Fahrzeuge raus. Wenn das Fahrzeug gerade fliegt, erhalten die Einheiten Fallschirme. Im Gegensatz zum Vanilla auswurf werden die Einheiten nacheinander aussteigen (nicht alle sofort, sondern mit einer kurzen Verzögerung dazwischen).</German>
```

Comment:
Vanilla auswurf?

Reply:
Not sure aobut that one, i would translate "eject" to "Auswurf" or "Rauswruf".

Reply:
maybe `Im Gegensatz zum "Vanilla" sitzen Einheiten nacheinander ab`